src/components/Claim/Link/Initial.view.tsx

```
@@ -351,9 +374,7 @@ export const InitialClaimLinkView = ({
                 return undefined
             }
 
-            const tokenAmount = Math.floor(
-                Number(claimLinkData.tokenAmount) * Math.pow(10, claimLinkData.tokenDecimals)
-            ).toString()
+            const tokenAmount: BigInt = parseUnits(claimLinkData.tokenAmount, claimLinkData.tokenDecimals)
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)